src/test/java/com/amazon/ion/conformance/expectations.kt

```diff
@@ -59,7 +60,7 @@ private fun IonReader.walk(): List<String> {
     while (true) {
         next()
         val currentType = type
-        if (type == null) {
+        if (currentType == null) {
```

Comment:
🗺️ This is a minor drive-by improvement that stops the IDE from complaining later on that `currentType` could be `null`.